lib/generators/my_zipcode_gem/templates/county_model.rb

```diff
@@ -1,16 +1,16 @@
 require 'memoist'
+
 class County < ActiveRecord::Base
   extend Memoist
-  attr_accessible :state_id, :region_id, :abbr, :name, :count_seat
-
   belongs_to :state
   has_many :zipcodes
-  
-  validates :name, :uniqueness => {:scope => :state_id, :case_sensitive => false}, :presence => true
-  
-  scope :without_zipcodes, joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL")
-  scope :without_state, where("state_id IS NULL")
-  
+
+  validates :name, uniqueness: { scope: :state_id, case_sensitive: false },
+                   presence: true
+
+  scope :without_zipcodes, -> { joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL") }
```

Comment:
Someday I'd like to work out the Arel for this, but I have too much to do for today.